pages/science/reactors/customreactor.rst

```diff
@@ -0,0 +1,49 @@
+.. title: Custom Reactors
+.. has_math: true
+
+.. jumbotron::
+
+   .. raw:: html
+
+      <h1 class="display-3">Custom Reactors</h1>
+
+   .. class:: lead
+
+      This page defines and links to an example of a Custom Reactor.
+
+Custom Reactors
+***************
+
+If the governing 
+equations must be entirely replaced or an integrator besides CVODES is 
+desired, a `Custom Reactor </science/reactors/customreactor.html>`__
+may be the most appropriate Reactor class to use. In this situation, 
```

Comment:
To help distinguish between this approach and the `ExtensibleReactor` option, maybe we should just refer to this as "using an external integrator"? At this point, I would hope that the `ExtensibleReactor` approach covers most use cases and that people shouldn't need to fall back to this option very often.